src/CattyTests/Bricks/RepeatBrickTests.swift

```diff
@@ -73,4 +73,27 @@ final class RepeatBrickTests: XCTestCase {
         XCTAssertEqual(0, repeatBrick.repetitions)
         XCTAssertNil(repeatBrick.maxRepetitions)
     }
+
+    func testMutableCopy() {
+        let interpreter = FormulaManager(sceneSize: CGSize.zero)
+
+        let brick = RepeatBrick()
+        let script = Script()
+        let object = SpriteObject()
+
+        script.object = object
+        brick.script = script
+        brick.timesToRepeat = Formula(double: 2)
+
+        XCTAssertTrue(brick.checkCondition(formulaInterpreter: interpreter))
+
+        let copiedBrick: RepeatBrick = brick.mutableCopy(with: CBMutableCopyContext()) as! RepeatBrick
+
+        XCTAssertTrue(brick.isEqual(to: copiedBrick))
+        XCTAssertFalse(brick === copiedBrick)
+        XCTAssertEqual(brick.maxRepetitions, copiedBrick.maxRepetitions)
+        XCTAssertEqual(copiedBrick.repetitions, 1)
+        XCTAssertTrue(brick.maxRepetitions === copiedBrick.maxRepetitions)
```

Comment:
The _maxRepetitions_ and _repetitions_ instances should not be the same references.

Reply:
Thanks for confirming that! But the test is failing. Earlier, I had removed the mutableCopy override from RepeatBrick, which I added back and tested too. But still the test is failing, i.e. references for maxRepetitions are getting same regardless there's an override or not. 
In fact, overriding method which I had removed had a following statement before returning the brick...
brick.maxRepetitions = self.maxRepetitions;
I think this statement will actually make reference of maxRepetitions same.

Reply:
Thanks! I think bringing back that method is a good idea. Please try to initialize them with zero, this will actually be more accurate since the newly copied Brick should have its own, independent "repetition logic".

Reply:
I think what I understand is I should be comparing timesToRepeat instead of maxRepetitions and repetitions.